pkg/descriptions.json

```diff
@@ -1962,6 +1962,13 @@
       "System.Text.Encodings.Web.JavaScriptEncoder"
     ]
   },
+  {
+    "Name": "System.Text.Json",
+    "Description": "Low allocation UTF-8 JSON reader/writer.",
```

Comment:
How about something with more symmetry with the XmlReader description such as "Provides the System.Text.Json.Utf8JsonReader type, that provides fast, noncached, forward-only access to JSON data."

Reply:
For now, that works, but eventually the library will contain components outside of reader/writer. We can adjust it then.